### DISCUSSION TITLE
Use tcconfig as a python module

### DISCUSSION THREAD

**Discussion by chomes** [thombashi/tcconfig#173] [Q&A] [answered]
Hi there, I like this package, and it seems very easy to use as a cli tool however I have a purpose/need for using it as a python module.  I've looked at the documentation, and it seems like it's all CLI based (if I'm missing a module section could I please be re-directed).  Is there any documentation for how to use it as a python module or would the best case scenario simply be to view the tests to get an idea how to use it in this manner?

**Comment by thombashi** [comment, marked as answer]
Unfortunately, there is no documentation of library usage.
You can find some test cases at [here](https://github.com/thombashi/tcconfig/blob/master/test/test_traffic_control.py).
Please note. Backward compatibility may break without notice for that use cases
